src/main/java/Parser.java

```diff
@@ -0,0 +1,170 @@
+                    LocalDate xx = LocalDate.parse(temp);
+                    String f = xx.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
+
+                    Task task = new Deadline(part1.substring(9), f);
+                    holder = holder + task + "\n";
+                    new TaskList().addToList(arrayList, task);
+                } else {
+                    Task task = new Deadline(part1.substring(9), part2.substring(3));
+                    holder = holder + task + "\n";
+                    new TaskList().addToList(arrayList, task);
+                }
+
+                holder = holder.concat("Now you have " + arrayList.size() + " task(s) in the list");
+
+
+            } else if (input.contains("event ")) {
+                holder = "Got it. I've added this task: "+ "\n";
+                String[] parts = input.split("/",2);
+                String part1 = parts[0];
+                String part2 = parts[1];
+
+                // e.g. event project meeting /at 02/12/2019 2-4pm
+                if (part2.contains("/")) {
+                    String dateString = part2.substring(3,13);
+                    String timeString = part2.substring(13);
+                    String temp = dateString.substring(6) + "-" + dateString.substring(3, 5) + "-" + dateString.substring(0, 2);
+                    LocalDate xx = LocalDate.parse(temp);
+                    String f = xx.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + timeString ;
+
+                    Task task = new Event(part1.substring(6), f);
+                    holder = holder + task + "\n";
+                    new TaskList().addToList(arrayList, task);
+                } else {
+                    Task task = new Event(part1.substring(6), part2.substring(3));
+                    holder = holder + task + "\n";
+                    new TaskList().addToList(arrayList, task);
+                }
+                holder = holder.concat("Now you have " + arrayList.size() + " task(s) in the list");
+
+
+            }  else if (input.contains("delete ")) {
+
+                int deletedNumber = Integer.parseInt(input.substring(7));
+                try {
+                    if (deletedNumber > arrayList.size()) {
+                        throw new DukeException("☹ OOPS!!! Sorry item no found :-(");
+                    } else {
+                        holder = "Noted. I've removed this task: "+ "\n";
+                        new TaskList().deleteFromList(arrayList, deletedNumber);
+                        holder = holder.concat("Now you have " + arrayList.size() + " task(s) in the list");
+
+                    }
+                } catch (DukeException e) {
+                    return e.getMessage();
+                }
+
+            } else if (input.contains("find ")) {
+                holder = "Here are the matching tasks in your list: \n";
+                ArrayList<Task> tt = new ArrayList<>();
+                int counterOne = 1;
```

Comment:
I noticed that you are using 2 counters so you named them counterOne and counterTwo. I like the fact that you try to differentiate them, but maybe it would be more intuitive to name them something else?